docs/wiki/feature/irlight.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: wiki
+title: IR Lights
+component: irlight
+description: IR flashlights and multi function lasers.
+group: feature
+category: equipment
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 16
+  patch: 0
+---
+
+## 1. Overview
+
+### 1.1 DBAL-A3
+The DBAL-A3 is a multi function laser and can be run in the following modes:
```

Comment:
```suggestion

The DBAL-A3 is a multi function laser and can be run in the following modes:
```

Reply:
I 100% agree - but this is how it is in all the other docs.

Reply:
I know, and that should be fixed. No reason to continue it. :P

Reply:
Fiiiine